package.json

```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "changelog42": "0.9.0",
     "eslint": "2.11.0",
+    "eslint-rules": "https://github.com/magora-labs/eslint-rules.git#v1.0.0",
```

Comment:
LGTM, however: why not use the normal versioning schema instead of using a branch? You could publish the module on npm, I don't see why you shouldn't.
Then we would simply add 
```
...
"eslint-rules": "1.0.0",
...
```